_content/docs/contributing.md

```diff
@@ -62,12 +62,12 @@ This method requires a certain amount of technical knowledge but is a lot more f
 
 #### Writing a good commit message
 
-Using git we can see _what_ changed in a particular commit, only a good commit message can tell us _why_. This is very important for the future when other editors will be updating your work. In the case of show data, they'll need to know your sources and how credible information you've gathered is.
+Using git we can see _what_ changed in a particular commit, but a good commit message can tell us _why_. This is very important for the future when other editors will be updating your work. In the case of show data, they'll need to know your sources and how credible the information you've gathered is.
```

Comment:
think this needs to be `but only`. the commit and the message are different things.